apps/web/src/views/Kyc/index.tsx

```diff
@@ -134,7 +136,7 @@ export const Kyc: React.FC = () => {
             <H1 as="h1" color={tokens.colors.dark.secondary} scale="xxl">
               KYC
             </H1>
-            <H2 color="#F4EEFF">Verify your identity</H2>
+            <H2 color="#F4EEFF"{t('>Verify your identity')}</H2>
```

Comment:
```suggestion
            <H2 color="#F4EEFF">{t('Verify your identity')}</H2>
```

Reply:
Nice catch. Thanks. I created a VSCode snippet that when I select a piece of texts, hit Ctrl + T, the highlighted text will be wrapped with {t()}. So in this case, my selection was offset a bit.